scalars.Rmd

```diff
@@ -56,4 +56,283 @@ $$
 \bar{a} {+=} \bar{c}
 \qquad
 \bar{b} {+=} -\bar{c}
+$$
+
+## Multiplication
+
+$$
+c = a\cdot b
+$$
+
+### Derivatives {-}
+
+$$
+\frac{\partial}{\partial a} c = b
+\qquad
+\frac{\partial}{\partial b} c = a
+$$
+
+
+### Tangent {-}
+
+$$
+\dot{c} = \dot{a}\cdot b + \dot{b}\cdot a
+$$
+
+### Adjoints {-}
+
+$$
+\bar{a}\ {+=} \ \bar{c}\cdot b
+\qquad
+\bar{b}\ {+=} \ \bar{c}\cdot a
+$$
+
+## Division
+
+$$
+c = \frac{a}{b}
+$$
+
+### Derivatives {-}
+
+$$
+\frac{\partial}{\partial a} c = \frac{1}{b}
+\qquad
+\frac{\partial}{\partial b} c = -\frac{a}{b^2}
+$$
+
+
+### Tangent {-}
+
+$$
+\dot{c} = \frac{\dot{a}}{b}-\frac{a\cdot\dot{b}}{b^2}
+$$
+
+### Adjoints {-}
+
+$$
+\bar{a}\ {+=} \ \frac{\bar{c}}{b}
+\qquad
+\bar{b}\ {+=} \ -\frac{\bar{c}\cdot a}{b^2}
+$$
+
+## Exponent
```

Comment:
"Exponential" is better---it's the name of the function.

I'd prefer to consistently use `\exp(x)` rather than `e^x` because the former's better in complex equations.